index.html

```diff
@@ -7,21 +7,48 @@
   <title>Quiz App</title>
   <link rel="stylesheet" href="styles.css">
   <link rel="icon" href="./assets/favicon_io/favicon.ico" type="image/x-icon">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300..700&family=Delius&family=Mystery+Quest&family=Neucha&family=Oldenburg&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/4.1.1/animate.min.css" />
```

Comment:
Good idea to use some external fonts and animations! Well done on making them work.